### PR TITLE
Data review

### PR DESCRIPTION
**Pull request by abyerly2**

### CODE REVIEW

**Comment by abyerly2** [comment]
@danyaljj Pushed my task updates

File: tasks/ATOMIC - NL Rephrase 16/batch.csv [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿Title,Description,Keywords,ObjectUseId1,Object1,Use1,ObjectUseId2,Object2,Use2,ObjectUseId3,Object3,Use3,Answer.ObjectUseId1,Answer.ObjectUseId2,Answer.ObjectUseId3,Answer.feedback,Answer.nl_phrase1,Answer.nl_phrase2,Answer.nl_phrase3
+Title,Description,Keywords,ObjectUseId1,Object1,Use1,ObjectUseId2,Object2,Use2,ObjectUseId3,Object3,Use3,Answer.ObjectUseId1,Answer.ObjectUseId2,Answer.ObjectUseId3,Answer.feedback,Answer.nl_phrase1,Answer.nl_phrase2,Answer.nl_phrase3
```
Comment (yeganehkordi) [inline tasks/ATOMIC - NL Rephrase 16/batch.csv]: Let's drop `Answer.ObjectUseId` columns in ATOMIC tasks

File: tasks/ATOMIC - Required Objects (Sequence) 9/batch.csv [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+Title,Description,Keywords,SequenceId,Beginning,Middle,End,Answer.SequenceId,Answer.beginning-obj1,Answer.beginning-obj2,Answer.beginning-obj3,Answer.beginning-obj4,Answer.beginning-req-obj-na,Answer.beginning-use1,Answer.beginning-use2,Answer.beginning-use3,Answer.beginning-use4,Answer.beginning-y-obj1,Answer.beginning-y-obj2,Answer.beginning-y-obj3,Answer.beginning-y-req-obj-na,Answer.beginning-y-use1,Answer.beginning-y-use2,Answer.beginning-y-use3,Answer.end-obj1,Answer.end-obj2,Answer.end-obj3,Answer.end-obj4,Answer.end-req-obj-na,Answer.end-use1,Answer.end-use2,Answer.end-use3,Answer.end-use4,Answer.end-y-obj1,Answer.end-y-obj2,Answer.end-y-obj3,Answer.end-y-req-obj-na,Answer.end-y-use1,Answer.end-y-use2,Answer.end-y-use3,Answer.feedback,Answer.middle-obj1,Answer.middle-obj2,Answer.middle-obj3,Answer.middle-obj4,Answer.middle-req-obj-na,Answer.middle-use1,Answer.middle-use2,Answer.middle-use3,Answer.middle-use4,Answer.middle-y-obj1,Answer.middle-y-obj2,Answer.middle-y-obj3,Answer.middle-y-req-obj-na,Answer.middle-y-use1,Answer.middle-y-use2,Answer.middle-y-use3
```
Comment (yeganehkordi) [inline tasks/ATOMIC - Required Objects (Sequence) 9/batch.csv]: I think we can drop `Answer.feedback` and incomplete rows

File: tasks/ATOMIC KB Completion 2.2/batch.csv
Comment (yeganehkordi) [inline tasks/ATOMIC KB Completion 2.2/batch.csv:1]: Can you please take a look at its HTML? I think some of the columns such as score or relation shouldn't be an Answer field.

File: tasks/Abductive Reasoning 11/batch.csv
Comment (yeganehkordi) [inline tasks/Abductive Reasoning 11/batch.csv:1]: We can drop `Answer.feedback` here and in many of the following tasks.

File: tasks/Annotate WaNLI 23/batch.csv
Comment (yeganehkordi) [inline tasks/Annotate WaNLI 23/batch.csv:1]: Please check the HTML file for this task too. I doubt if premise and hypothesis columns should be answer.

File: tasks/Annotation subj_obj/batch.csv [hunk 3]
```diff
@@ -1,4 +1,4 @@
...
 Identify subjects for given verbs in English sentences.,You will be shown a list of English sentences each containing a particular verb. You will be asked to click on the word that is the subject of that verb.,"verbs, subjects, language, grammar",love,loving,loved,Shelties love to run in wide-open areas .,"Ozzy , Mom loves your stuff .","Our heartfelt condolences to all his family , friends and loved ones .",That night Elsie tells Mary how she did not love Sir William but did not mind him .,Duane Allman loved the area and frequently returned to the Shoals for session work throughout his life .,"But since the two rarely come together , anyone compelled to choose will find greater security in being feared than in being loved .","Jacob loves Rachel more than Leah , and Leah feels hated .","Disgusted by the arrangement , Trippy leaves home to fight in the Lafayette Escadrille in World War I. Job loves Fanny , but she all but ignores him .","Despite his austerity , John was loved .","As Creed is announced the winner by split decision , Adrian and Rocky embrace , and Rocky tells her that he loves her .",Mom,Adrian,,Jacob,John,Trippy,Elsie,two,,Duane,Shelties,no,,,,,,,,
```
Comment (yeganehkordi) [inline tasks/Annotation subj_obj/batch.csv]: Is this row incomplete?

File: tasks/COMET2020 ATOMIC Inference Vp 5/batch.csv [hunk 4]
```diff
@@ -1,4 +1,4 @@
...
...reasoning",PersonX begs PersonY's parents for one,PersonX does not beg PersonY 's parents for one,rich_people,well-off,well-to-do,wealthy,woeful,not whingeing,discouraged,despairing,pessimistic,abundant,nifty,privileged,not pathetic,wretched,prosperous,exalted,not happy that persony got something they wanted,not embarassed,ungrateful,//,//,//,//,//,//,//,//,//,//,//,not good they got something they wanted,not pressured,not embarassed,//,//,//,//,//,//,//,//,//,//,//,//,no help with personY,personY not to have somethiing,//,//,//,//,//,//,//,//,//,//,//,//,//,to not get dog food,to not get a dog,//,//,//,//,//,//,//,//,//,//,//,//,//,to not help out,to not go to the vet,//,//,//,//,//,//,//,//,//,//,//,//,//,to not muster courage for it,to not decide how to speak to them,//,//,//,//,//,//,//,//,//,//,//,//,//,doesn't cry,doesn't speak to beg .,heart rate doesn't increase with desire .,doesn't cajole,doesn't plead,//,//,//,//,//,//,//,//,//,//,doesn't decide whether or not to give anything to personX .,doesn't spend money on personX .,no deny,no cede,no grant,//,//,//,//,//,//,//,//,//,//,{},on,,on,on,on,,,,{},{},{},{},,28.574,,,on,,on,on,,,,,{},{},{},{},,on,,,,on,,,,,,,,{},{},{},{},
```
Comment (yeganehkordi) [inline tasks/COMET2020 ATOMIC Inference Vp 5/batch.csv]: There are lots of `//` and `{}` in this file. Is the annotation okay?

File: tasks/Coherence Evaluation/batch.csv [hunk 5]
```diff
@@ -1,4 +1,4 @@
-﻿Title,Description,Keywords,pair11,pair12,answer1,pair21,pair22,answer2,pair31,pair32,answer3,pair41,pair42,answer4,pair51,pair52,answer5,pair61,pair62,answer6,pair71,pair72,answer7,pair81,pair82,answer8,pair91,pair92,answer9,pair101,pair102,answer10,pair111,pair112,answer11,pair121,pair122,answer12,Answer.answer1,Answer.answer10,Answer.answer11,Answer.answer12,Answer.answer2,Answer.answer3,Answer.answer4,Answer.answer5,Answer.answer6,Answer.answer7,Answer.answer8,Answer.answer9,Answer.comment
+Title,Description,Keywords,pair11,pair12,answer1,pair21,pair22,answer2,pair31,pair32,answer3,pair41,pair42,answer4,pair51,pair52,answer5,pair61,pair62,answer6,pair71,pair72,answer7,pair81,pair82,answer8,pair91,pair92,answer9,pair101,pair102,answer10,pair111,pair112,answer11,pair121,pair122,answer12,Answer.answer1,Answer.answer10,Answer.answer11,Answer.answer12,Answer.answer2,Answer.answer3,Answer.answer4,Answer.answer5,Answer.answer6,Answer.answer7,Answer.answer8,Answer.answer9,Answer.comment
```
Comment (yeganehkordi) [inline tasks/Coherence Evaluation/batch.csv]: We can drop `Answer.comment`

**Review by yeganehkordi — COMMENTED** [review]
Adam, I reviewed the first 13 tasks and I think in most of the tasks we need to drop the comment column. 
Also, it will be great if you can take a look at the HTML files to see if they look good to you.

File: tasks/Annotation subj_obj/batch.csv (hunk 3 above)
Comment (abyerly2) [reply tasks/Annotation subj_obj/batch.csv]: No but the format is a bit clunky I think - basically the user needs to either (1) enter some text in a text box (these are the Answer.sentXsubject fields), or (2) click a checkbox indicating no subject is present (these are the Answer.sentXbox fields). So rows will have entries in either Answer.sentXsubject or Answer.sentXbox, but should not have both.

File: tasks/Human evaluation - quals/batch.csv [hunk 6]
```diff
@@ -1,121 +1,121 @@
...
-Indicate whether dialogue is steering in a positive way or not.,Indicate whether dialogue is steering in a positive way or not.,"dialogue, reasoning",I'm going to leave an 18-cent tip and get the waitress fired.,Which menu did you order? It’s nice to have family dinner time.,That would put her in a difficult situation. Why would you leave such a small tip and hope to get a waitress fired?,r2,195.009,tie,{},r2,r2,r1
-Indicate whether dialogue is steering in a positive way or not.,Indicate whether dialogue is steering in a positive way or not.,"dialogue, reasoning",I'm going to leave an 18-cent tip and get the waitress fired.,Which menu did you order? It’s nice to have family dinner time.,That would put her in a difficult situation. Why would you leave such a small tip and hope to get a waitress fired?,r2,40.877,tie,{},r2,r2,tie
+Title,Description,Keywords,context1,response1,response2,Answer.coherency,Answer.ee,Answer.engaged,Answer.overall,Answer.prosociality,Answer.respect
```
Comment (yeganehkordi) [inline tasks/Human evaluation - quals/batch.csv]: Please drop `Answer.ee` in the tasks.

File: tasks/ROT Details [m=50] rocstories - 0 - 99/batch.csv
Comment (yeganehkordi) [inline tasks/ROT Details [m=50] rocstories - 0 - 99/batch.csv:1]: I think we can drop `Answer.te` in this task.

File: tasks/Transcribe Meeting Speech for Extractive Summarization firstpass/batch.csv
Comment (yeganehkordi) [inline tasks/Transcribe Meeting Speech for Extractive Summarization firstpass/batch.csv:1]: Do we have the audios of this task?

File: tasks/new accept (dev)/batch.csv [hunk 7]
```diff
@@ -1,751 +1,501 @@
...
-Do you agree with the moral judgment?,"Given a statement and a moral judgment, tell us if you agree or disagree with the judgment.","annotation, moral judgment",Jim sets up a charity to send underprivileged youth to college,want to stay home in your pajamas instead of going out all the time,"look at a map if you're lost, when we were glad to have the map because we got lost","in the context that Alan is a 4th grader who is excited about a Christmas party that he will be going to tonight, and Alan wants s
```
Comment (yeganehkordi) [inline tasks/new accept (dev)/batch.csv]: Please drop `Answer.ee` column in this task.

File: tasks/evaluate_e2e 5/batch.csv [hunk 8]
```diff
@@ -1,601 +1,501 @@
...
-Evaluate the quality of the text considering the given keywords,Evaluate the quality of the text considering the given keywords,"keywords, generation",name : The Phoenix | Type : pub | food : French | price : £ 20 - 25 | customer rating : 3 out of 5 | area : city centre | family friendly : no | near : Café Sicilia,The Phoenix is a French pub located in the city centre near Café Sicilia. The price range is £ 20 - 25 and the customer rating is 3 out of 5. It is not kid friendly.,a_star,4,good,2,9.107
-Evaluate the quality of the text considering the given keywords,Evaluate the quality of the text considering the given keywords,"keywords, generation",name : The Phoenix | Type : pub | food : French | price : £ 20 - 25 | customer rating : 3 out of 5 | area : city centre | family friendly : no | near : Café Sicilia,The Phoenix is a French pub located in the city centre near Café Sicilia. The price range is £ 20 - 25 and the customer rating is 3 out of 5. I
```
Comment (yeganehkordi) [inline tasks/evaluate_e2e 5/batch.csv]: Please drop `Answer.tm` column.

File: tasks/wikiHow Step Ordering/batch.csv
Comment (yeganehkordi) [inline tasks/wikiHow Step Ordering/batch.csv:1]: Can we replace the annotators' responses with gold labels in this task?

**Review by yeganehkordi — COMMENTED** [review]
Thank you!
There are just a few comments and then we are good to merge.

File: tasks/Transcribe Meeting Speech for Extractive Summarization firstpass/batch.csv
Comment (abyerly2) [reply tasks/Transcribe Meeting Speech for Extractive Summarization firstpass/batch.csv:1]: After looking at it more closely, we may need to drop this task - all of the audio files are "hidden_for_privacy_reasons"

Comment (yeganehkordi) [reply tasks/Transcribe Meeting Speech for Extractive Summarization firstpass/batch.csv:1]: Let's drop it

File: tasks/ROT Details [m=50] rocstories - 0 - 99/batch.csv
Comment (yeganehkordi) [reply tasks/ROT Details [m=50] rocstories - 0 - 99/batch.csv:1]: I'm also not sure about `Answer.situation-id`, `Answer.situation-short-id`, `Answer.rot-id`, and `rot-id`. Can models generate these ids?

**Pull request merged**